api/src/main/java/io/strimzi/api/kafka/model/KafkaUserQuotas.java

```diff
@@ -34,6 +34,17 @@
     private Integer producerByteRate;
     private Integer consumerByteRate;
     private Integer requestPercentage;
+    private Double controllerMutationRate;
+
+    @Description(" The quota on rate at which mutations are accepted for the create topics request, the create partitions request and the delete topics request. The rate is accumulated by the number of partitions created or deleted.")
+    @Minimum(0)
```

Comment:
The default value for this is the KIP is `Long.MaxValue`, won't setting this to zero prevent any changes?